de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/inventory/process/M_Inventory_CreateCountLinesFromHU.java

```diff
@@ -108,6 +109,8 @@ private I_M_Inventory getInventory()
 			huQueryBuilder.addOnlyWithProductId(productId);
 		}
 
+		huQueryBuilder.addHUStatusToExclude(X_M_HU.HUSTATUS_Planning);
```

Comment:
AFAIU this is not really making sure to allow only "Active" and "Issued" HUs in inventorylines..
in particular, how do we make sure that shipped or destroyed HUs are not added?

Apart from that, @metas-dh : 
does it even make sense to also exclude e.g. "picked" HUs? 
Afaiu, as long as they are not planned, shipped or destroyed, shouldn't they be part of the inventory?

Reply:
Who knows how exactly should be?

Reply:
loud thinking: why not considering Active, Issued and Picked?
i.e. de.metas.handlingunits.IHUStatusBL.getQtyOnHandStatuses()

@cristinamghita  like Tobi said, please use `addHUStatusesToInclude`.